mirest/stylist-api
# #165113834 Add user type field

#### What does this PR do?
 - It creates the user  `model`

#### Description of Task to be completed?
- Add user type field
- Write tests for the field
#### How should this be manually tested?
- Pull the branch `git fetch origin ch-add-user-type-165113834` 
- Check it out `git checkout ch-add-user-type-165113834`
- Run `coverage run --source='apps' manage.py test`

#### Pivotal Tracker Story
- [#165113834](https://www.pivotaltracker.com/story/show/165113834)
